web/static/frontend/css/machines-layout.scss

```diff
@@ -64,3 +63,7 @@
     }
   }
 }
+
+.inactive-machine-list {
+  // TODO
+}
```

Comment:
Files should end with a trailing newline

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#finalnewline)